transpiler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "transpiler",
+    "version": "1.0.0",
+    "description": "Transpiles site apps TSX to browser JS via stdin/stdout",
+    "main": "transpile.mjs",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "build": "tsc -b && node build.mjs",
+        "start:dist": "node dist/index.js",
+        "start": "npm run build && npm run start:dist"
+    },
+    "author": "PostHog <hey@posthog.com>",
```

Comment:
that is not a valid email anymore - as in it gets an autoreply of use some other email depending on your need

Reply:
hmm... I removed the email completely.